cloudevents/work/lister.go

```diff
@@ -0,0 +1,28 @@
+package work
+
+import (
+	"k8s.io/client-go/tools/cache"
+
+	"open-cluster-management.io/api/cloudevents/generic/types"
+	workv1 "open-cluster-management.io/api/work/v1"
+)
+
+// ManifestWorkLister list the ManifestWorks from a ManifestWorkInformer's local cache.
+type ManifestWorkLister struct {
```

Comment:
how does this lister get initiated?

Reply:
when we create the work client for cloud events, we use inform store to initiate it

Reply:
if it is from informer, why we not use lister directly but using cache.store?

Reply:
yeah, a lister can be used here, I will use lister instead of store